### ISSUE TITLE
Identity filter increments large integer values unexpectedly

### ISSUE THREAD

**Issue by wdh** [stedolan/jq#2131]
Input:
```$ echo '{"variantId":24092600057481699}' | jq '.'```

Actual output:
```
{
  "variantId": 24092600057481700
}
```

Expected output:
```
{
  "variantId": 24092600057481699
}
```

Notes:
* The issue seems to be related to the size of the integer - it does not happen if the leading digit is removed.
* Expected output is based on `man jq`, which says it should be unchanged:
```
BASIC FILTERS
   Identity: .
       The absolute simplest filter is . . This is a filter that takes its input and produces it unchanged as output. That is, this is the identity operator.
```

Environment:
* MacOS Mojave 10.14.6 (18G4032)
* jq 1.6 (installed via Homebrew)
```
$ brew info jq
jq: stable 1.6 (bottled), HEAD
Lightweight and flexible command-line JSON processor
https://stedolan.github.io/jq/
/usr/local/Cellar/jq/1.6 (18 files, 1MB) *
  Poured from bottle on 2020-01-20 at 11:22:05
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/jq.rb
==> Dependencies
Required: oniguruma ✔
==> Options
--HEAD
        Install HEAD version
==> Analytics
install: 37,670 (30 days), 122,484 (90 days), 456,202 (365 days)
install-on-request: 36,632 (30 days), 119,189 (90 days), 442,258 (365 days)
build-error: 0 (30 days)
```

**Comment by itchyny** [comment]
This is a frequently asked question (#545 #1652 #1959, #2109) and is already fixed in the master HEAD (implemented by #1752).
```
 % jq --version
jq-master-50a7022
 % echo '{"variantId":24092600057481699}' | jq '.'
{
  "variantId": 24092600057481699
}
```

**Comment by wdh** [comment]
Thanks for the info. Built from master per the README.md and confirm WFM:
```
$ ~/bin/jq --version
jq-1.6-124-gccc79e5-dirty
[wdh@bat ~]$ echo '{"variantId":24092600057481699}' | ~/bin/jq .
{
  "variantId": 24092600057481699
}
```
